datafusion/common/src/error.rs

```diff
@@ -602,6 +607,9 @@ impl DataFusionError {
         DiagnosticsIterator { head: self }.next()
     }
 
+    /// Return an iterator this [`DataFusionError`] and any other
```

Comment:
Perhaps it should be "Return an iterator over this ..."?

Reply:
in 1da42a2b8